src/peer/dataConnection.js

```diff
@@ -137,7 +137,7 @@ class DataConnection extends Connection {
 
     this._dc.onclose = () => {
       logger.log('DataChannel closed for:', this.id);
-      this.close();
+      this.close(true);
```

Comment:
👀

Reply:
I'm wondering it's OK to set this `true`..

Reply:
What are you concerned about?

Reply:
I see, we might not change this behavior. I'll fix this.